_data/docs.yml

```diff
@@ -9,9 +9,12 @@
   link: user-tutorials
   docs:
     - user-tutorials/faceted-search
+    - user-tutorials/language-support
     - user-tutorials/launch
+    - user-tutorials/dnastack-launch-with
```

Comment:
The link for user-tutorials/dnastack-launch-with does not match that of _docs/dnastack-launch-with.md